### ISSUE TITLE
Improvement - Couples Match

### ISSUE THREAD

**Issue by VP-Communications** [cfms-website/cfms-website.github.io#63]
https://www.cfms.org/resources/couples-match-app/

Needs to be updated.

**Comment by pyarmak** [comment]
Oh yeah, forgot about this one. Yeah, it's completely broken.

**Comment by Threatsignal** [comment]
@VP-Communications This issue is taking longer than the estimate Pavel and I originally thought.
It is more than just including angular dependencies. The code that was written for the couples match making app did not contain a package.json file which linked the dependencies it used to their version numbers. 
Since angular is a growing technology and same with a lot of the dependencies used. They have all had major updates and therefore the code that was used at the time this app was written no longer works with these newer versions.
Because of this, I have had to dig through the git commit logs and find the code that was used for this project from before. So far I have spent roughly 2 hours on this compared to our 1 hour estimate. 
I'm going to need at least a couple more hours to hunt down all dependencies and have their version numbers coincide with one another.

Once I have done this I'm going to write a package.json file and keep these modules separate from the actual website so it will be a separated module from the rest of the website to help avoid this issue in the future.
